ts/src/pro/probit.ts

```diff
@@ -314,6 +300,10 @@ export default class probit extends probitRest {
         client.resolve (stored, messageHash);
     }
 
+    parseTrade (rawTrade: Dict, market: Market = undefined): Trade {
+        return super.parseTrade (rawTrade, market);
+    }
+
```

Comment:
why this? why don't we just call parseTrade then?

Reply:
I was planning to add some additional parsing in `ws` parsing here, but havent added. maybe yes, i should remove these lines now.